docs/generate_example_images.py

```diff
@@ -27,53 +27,57 @@ def main():
     img_t = img_t.float() / 255.0
-    img_t = img_t.repeat(2, 1, 1, 1)
 
     # TODO: make this more generic for modules out of kornia.augmentation
     # Dictionary containing the transforms to generate the sample images:
     # Key: Name of the transform class.
-    # Value: The default parameters to use.
+    # Value: (parameters, num_samples, seed)
     augmentations_list: dict = {
-        "ColorJitter": (0.1, 0.1, 0.1, 0.1),
-        "RandomAffine": ((-15.0, 20.0),),
-        "RandomBoxBlur": ((7, 7),),
-        "RandomChannelShuffle": (),
-        "RandomErasing": ((0.4, 0.8), (0.3, 1 / 0.3)),
-        "RandomElasticTransform": ((63, 63), (32, 32), (1.0, 1.0)),
-        "RandomEqualize": (),
-        "RandomFisheye": (torch.tensor([-0.3, 0.3]), torch.tensor([-0.3, 0.3]), torch.tensor([0.9, 1.0])),
-        "RandomGrayscale": (),
-        "RandomGaussianBlur": ((3, 3), (0.1, 2.0)),
-        "RandomGaussianNoise": (0.0, 0.05),
-        "RandomHorizontalFlip": (),
-        "RandomInvert": (),
-        "RandomMotionBlur": (7, 35.0, 0.5),
-        "RandomPerspective": (0.2,),
-        "RandomPosterize": (3,),
-        "RandomResizedCrop": ((510, 1020), (3.0, 3.0), (2.0, 2.0)),
-        "RandomRotation": (45.0,),
-        "RandomSharpness": (1.0,),
-        "RandomSolarize": (0.1,),
-        "RandomVerticalFlip": (),
-        "RandomThinPlateSpline": (),
+        "ColorJitter": ((0.3, 0.3, 0.3, 0.3), 2, 2018),
+        "RandomAffine": (((-15.0, 20.0), (0.1, 0.1), (0.7, 1.3), 20), 2, 2019),
+        "RandomBoxBlur": (((7, 7),), 1, 2020),
+        "RandomChannelShuffle": ((), 1, 2020),
+        "RandomElasticTransform": (((63, 63), (32, 32), (2.0, 2.0)), 2, 2018),
+        "RandomEqualize": ((), 1, 2020),
+        "RandomErasing": (((0.2, 0.4), (0.3, 1 / 0.3)), 2, 2017),
+        "RandomFisheye": ((torch.tensor([-0.3, 0.3]), torch.tensor([-0.3, 0.3]), torch.tensor([0.9, 1.0])), 2, 2020),
+        "RandomGaussianBlur": (((3, 3), (0.1, 2.0)), 1, 2020),
+        "RandomGaussianNoise": ((0.0, 0.05), 1, 2020),
+        "RandomGrayscale": ((), 1, 2020),
+        "RandomHorizontalFlip": ((), 1, 2020),
+        "RandomInvert": ((), 1, 2020),
+        "RandomMotionBlur": ((7, 35.0, 0.5), 2, 2020),
+        "RandomPerspective": ((0.2,), 2, 2020),
+        "RandomPosterize": (((1, 4),), 2, 2016),
+        "RandomResizedCrop": ((img.shape[:2], (1.0, 2.0), (1.0, 2.0)), 2, 2020),
+        "RandomRotation": ((45.0,), 2, 2019),
+        "RandomSharpness": ((16.0,), 1, 2019),
+        "RandomSolarize": ((0.2, 0.2,), 2, 2019),
+        "RandomVerticalFlip": ((), 1, 2020),
+        "RandomThinPlateSpline": ((), 1, 2020),
     }
 
     # ITERATE OVER THE TRANSFORMS
 
-    for aug_name, args in augmentations_list.items():
+    for aug_name, (args, num_samples, seed) in augmentations_list.items():
+        img_in = img_t.repeat(num_samples, 1, 1, 1)
         # dynamically create the class instance
         cls = getattr(mod, aug_name)
         aug = cls(*args, p=1.0)
```

Comment:
@shijianjian the seed could be added directly here I guess

Reply:
We have had a discussion at some point about adding a seed param to the function. While it means to maintain multiple sub RNG states in order to not mess up the main thread. Not an easy add-on, I will therefore evaluate further with the current API.